local-cli/runIOS/runIOS.js

```diff
@@ -112,6 +112,24 @@ function runOnSimulator(xcodeProject, args, scheme) {
     if (!selectedSimulator) {
       throw new Error(`Could not find ${args.simulator} simulator`);
     }
+    
+    /**
+		 * Booting simulator through `xcrun simctl boot` will boot it in the `headless` mode
+		 * (running in the background).
+		 *
+		 * In order for user to see the app and the simulator itself, we have to make sure
+		 * that the Simulator.app is running.
+		 *
+		 * We also pass it `-CurrentDeviceUDID` so that when we launch it for the first time,
+		 * it will not boot the "default" device, but the one we set. If the app is already running,
+		 * this flag has no effect.
+		 */
+		child_process.execFileSync('open', [
```

Comment:
I believe `open -a Simulator` should launch the simulator as well, without specifying a full path. Although not sure if you're then able to pass it additional flags

Reply:
After further investigation, I think you should be able to use;

```
open -a Simulator --args -CurrentDeviceUDID ...
```

Reply:
Yeah, that is nice!